aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -234,7 +233,7 @@
         \\"sigopt_dev_id\\":\\"$SIGOPT_DEV_ID\\"
       }
     }" $(ec2metadata --instance-id) $(ec2metadata --instance-type) $(ec2metadata --public-hostname) $(ec2metadata --public-hostname) "${s3p#","}")
-  - /tmp/slack.sh "$bye_msg"
+  - curl -H "Authorization:Bearer $SLACK_TOKEN" -F file=@RunHealthAnalysis.txt -F initial_comment="$bye_msg" -F channels="$SLACK_CHANNEL" "https://slack.com/api/files.upload"  
```

Comment:
Have you tested this out? I hadn't used the initial_comment with a formatted string - I assume it's ok though

Reply:
yes @JustinPihony , You can check this screenshot
<img width="940" alt="Screenshot 2021-12-24 at 11 41 10 PM" src="https://user-images.githubusercontent.com/5759657/147368362-aff5099d-8f28-477a-a409-b52d3b46dd64.png">